applications/asset_tracker_v2/tests/gps_module/src/gps_module_test.c

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <unity.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <mock_modules_common.h>
+#include <mock_event_manager.h>
+#include <drivers/mock_gps.h>
+#include "app_module_event.h"
+#include "gps_module_event.h"
+#include "data_module_event.h"
+
+extern struct event_listener __event_listener_gps_module;
+
+#define GPS_MODULE_EVT_HANDLER(eh) __event_listener_gps_module.notification(eh)
+
+/* NMEA sample data, including CRC.
+ * Source: https://docs.novatel.com/OEM7/Content/Logs/GPGGA.htm
+ */
+#define GPS_NMEA_SAMPLE                                                        \
+	"$GPGGA,134658.00,5106.9792,N,11402.3003,W,2,09,1.0,1048.47,M,-16.27,M,08,AAAA*60"
+
+
+static char exp_nmea[NMEA_MAX_LEN];
+
+/* Dummy functions and objects. */
+#if defined(CONFIG_BOARD_NATIVE_POSIX) || defined(CONFIG_BOARD_QEMU_CORTEX_M3)
+void sys_reboot(int reason)
+{
+}
+#endif
+
+void *k_malloc(size_t size)
+{
+	return malloc(size);
+}
+
+/* Dummy structs to please linker. The EVENT_SUBSCRIBE macros in gps_module.c
+ * depend on these to exist. But since we are unit testing, we dont need
+ * these subscriptions and hence these structs can remain uninitialized.
+ */
+const struct event_type __event_type_gps_module_event;
+const struct event_type __event_type_app_module_event;
+const struct event_type __event_type_data_module_event;
+const struct event_type __event_type_util_module_event;
+
+/** Dummy functions and objects - End.  **/
+
+/* The following is required because unity is using a different main signature
+ * (returns int) and zephyr expects main to not return value.
+ */
+extern int unity_main(void);
+
+/* Variable to store the event handler function that the GPS module registers
+ * with the GPS driver through the gps_init() function. This can be used to
+ * simulate events from the GPS driver.
+ */
+static gps_event_handler_t gps_module_gps_evt_handler;
+
+/* Suite teardown finalizes with mandatory call to generic_suiteTearDown. */
+extern int generic_suiteTearDown(int num_failures);
+
+
+int test_suiteTearDown(int num_failures)
+{
+	return generic_suiteTearDown(num_failures);
+}
+
```

Comment:
Here as well, could we keep to only use a single newline? I see occurrences of both single and doubles in the file.

Reply:
I was using two new lines after each function. But I see that its inconsistent. I will revert to single new line.